darwin/utils/utils.py

```diff
@@ -993,13 +994,17 @@ def split_video_annotation(annotation: dt.AnnotationFile) -> List[dt.AnnotationF
         no ``frame_url`` attribute.
     """
     if not annotation.is_video:
-        raise AttributeError("this is not a video annotation")
-
-    if not annotation.frame_urls:
-        raise AttributeError("This Annotation has no frame urls")
-
+        raise AttributeError("This is not a video annotation")
+
+    if not annotation.frame_count and not annotation.frame_urls:
+        raise AttributeError("This Annotation has no frames")
+    urls = (
+        annotation.frame_urls
+        if annotation.frame_urls
+        else [None] * (annotation.frame_count or 1)
```

Comment:
Just a bit pythonic this way with ternary operator:
```suggestion
    urls = annotation.frame_urls or [None] * (annotation.frame_count or 1)
```